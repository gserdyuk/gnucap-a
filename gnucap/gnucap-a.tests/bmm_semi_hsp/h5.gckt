* parameters of V1 w/o quotes
'parameter test
.param a=1 b=2 c=3 d=4
v1 1 0 dc a ac b  pwl 0 5 0.1 1
r1 1 0 a+b
.print op v(1) i(v1)
.op
.print ac v(1) i(v1)
.ac dec 2 1 10
.end
