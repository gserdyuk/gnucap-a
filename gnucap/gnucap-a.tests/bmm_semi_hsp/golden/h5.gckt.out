Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* parameters of V1 w/o quotes 
#           v(1)       i(v1)     
 27.        1.        -0.33333   
#Freq       v(1)       i(v1)     
 1.         2.         0.66666   
 3.1623     2.         0.66666   
 10.        2.         0.66666   
